asyncpg_listen/listener.py

```diff
@@ -169,3 +178,32 @@ def _push(_: Any, __: Any, channel: Any, payload: Any) -> None:
             queue.put_nowait(Notification(channel, payload))
 
         return _push
+
+    async def _process_notification(self, handler: NotificationHandler, notification: NotificationOrTimeout) -> None:
+        if self._tracer is None:
+            self._tracer = opentelemetry.trace.get_tracer(__package__)
+        if self._meter is None:
+            self._meter = opentelemetry.metrics.get_meter(__package__)
+        if self._notification_histogram is None:
+            self._notification_histogram = self._meter.create_histogram("asyncpg_listener_notification")
```

Comment:
what if  move this three `iff` blocks to funciton like `_ensure_setup()  -> NamedTuple(tracecr, meter. notification)` or something like this? or make properties with backing fields?